### PR TITLE
v0.1.0-beta

### PR DESCRIPTION
**Pull request by brendandixon**
This pull request brings the ARM packages up to "beta" status. It addresses multiple issues. See the CHANGELOG.md for details.

Signed-off-by: Brendan Dixon <brendand@microsoft.com>

### CODE REVIEW

**Comment by brendandixon** [comment]
@ahmetalpbalkan @paulmey Here is the beta version of the ARM packages.

I tried to vendor our dependencies via godep, but ran into an issue. It complains it cannot find the package "gopkg.in/check.v1". Any ideas?

File: arm/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# CHANGELOG
+
+## v0.1.0-alpha
```
Comment (ahmetalpbalkan) [inline arm/CHANGELOG.md]: version ordering in this doc better be chronologically reverse order.

also please use line separator `------` between versions, I can't tell which title belongs to which version.

**Comment by ahmetalpbalkan** [comment]
LGTM.

**Comment by ahmetalpbalkan** [comment]
Taking LGTM back. Where did `Godeps/` go? Can't see them in the diff?

**Comment by brendandixon** [comment]
@ahmetalpbalkan I left them off until I figured out the Godep issue. I've updated the PR with the dependencies for the ARM packages only. I'll let you figure out the others. The problem is with the storage package.

**Comment by ahmetalpbalkan** [comment]
lgtm

**Pull request merged**